app/templates/partials/gtm.html

```diff
@@ -1,18 +1,16 @@
 <!-- Google Tag Manager -->
 <script nonce="{{ cspNonce }}">
-    function loadGTM() {
-        (function (w, d, s, l, i) {
-            w[l] = w[l] || []; w[l].push({
-                'gtm.start':
-                    new Date().getTime(), event: 'gtm.js'
-            }); var f = d.getElementsByTagName(s)[0],
-                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
-                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl + '&gtm_auth={{ gtm_auth }}&gtm_cookies_win=x'; f.parentNode.insertBefore(j, f);
-        })(window, document, 'script', 'dataLayer', '{{ gtm_cont_id }}');
-    }
-    var a = /^(.*)?\s*'usage':true\s*[^;]+(.*)?$/;
-    if (document.cookie.match(a)) {
-        loadGTM();
+    var allowTrackingCookies = /^(.*)?\s*'campaign':true\s*[^;]+(.*)?$/;
```

Comment:
Any reason you used `campaign` instead of `usage` like previously? It should also be `campaigns` 
```suggestion
    var allowTrackingCookies = /^(.*)?\s*'campaigns':true\s*[^;]+(.*)?$/;
```